File: Farhaan/Week-1 Problems/HTMLTAGS.cpp
#include <iostream>
#include <string>
using namespace std;
void check(string str,int last){
      int temp;
      for(int i = 2;i<last;i++){
            temp = str[i];
            if(temp>47 && temp<58)
                continue;
            if(temp<97|| temp>122){
                cout<<"Error"<<endl;
                return;
            }
        }
        cout<<"Success"<<endl;
}
int main(){
    string str;
    int t,last;
    cin>>t;
    while(t--){
       cin>>str;
       last = str.length()-1;
        if(str[0]!='<' || str[1]!='/' || str[last]!='>' || str.length()==3){
            cout<<"Error"<<endl;
            continue;
        }
        check(str,last);
    }
    return 0;
}

File: Jhanavi/09-09-2022/HTML_Tags.py
n = int(input())
an = "1234567890qwertyuiopasdfghjklzxcvbnm"
for i in range(n):
    string = input().strip()
    temp = string[2:len(string) - 1]
    flag = False
    if string[0:2] == '</' and string[-1] == '>' and len(temp):
        for c in temp:
            if c not in an:
                print('Error')
                flag = True
                break
        if not flag:
            print('Success')
    else:
        print('Error')
